src/views/javascripts/logic.js

```diff
@@ -0,0 +1,240 @@
+if (!localStorage.getItem('lastConnection')) {
+  localStorage.setItem('lastConnection', 'default');
+}
+
+const Task = () => {
+  const title = null;
+  const description = null;
+  const date = null;
+  const time = null;
+  const priority = null;
+  const note = null;
+  const status = null;
+  const id = null;
+
+  return {
+    title, description, date, time, priority, note, status, id,
+  };
+};
+
+function closeFrm(mainDiv) {
+  document.body.removeChild(mainDiv);
+}
+
+function closeTaskFrm() {
+  const main = document.getElementById('mainContainer');
+  const container = document.getElementsByClassName('task-popup');
+  main.removeChild(container[0]);
+}
```

Comment:
this is a sample of what we mean by manipulating the DOM inside a logic code
(You practically named this file `logic.js` yourself)